packages/ethereum-networks/lib/index.ts

```diff
@@ -93,6 +93,22 @@ const bsc_testnet: EthereumNetwork = {
   isTestnet: true
 }
 
+const polygon_mainnet: EthereumNetwork = {
+  name: 'matic_mainnet',
```

Comment:
why the difference between the name and the variable name?

Reply:
This one is fixed, the name should be polygon_mainnet.